SLURM_MEM_PER_CPU=8192
SLURM_NODEID=0
SLURM_TASK_PID=1038199
SLURM_PRIO_PROCESS=0
SLURM_SUBMIT_DIR=/users/ibarbier/AC-DC
SLURM_CPUS_PER_TASK=1
SLURM_PROCID=0
SLURM_JOB_GID=11000
SLURMD_NODENAME=dna001
SLURM_TASKS_PER_NODE=1
SLURM_NNODES=1
SLURM_JOB_NODELIST=dna001
SLURM_CLUSTER_NAME=curnagl
SLURM_NODELIST=dna001
SLURM_NTASKS=1
SLURM_JOB_CPUS_PER_NODE=1
SLURM_TOPOLOGY_ADDR=core.edge1.dna001
SLURM_WORKING_CLUSTER=curnagl:admin1:6817:9216:109
SLURM_JOB_NAME=sbatch_plot.sh
SLURM_JOBID=5983318
SLURM_CONF=/etc/slurm/slurm.conf
SLURM_NODE_ALIASES=(null)
SLURM_JOB_QOS=cpu-normal
SLURM_TOPOLOGY_ADDR_PATTERN=switch.switch.node
SLURM_CPUS_ON_NODE=1
SLURM_JOB_NUM_NODES=1
SLURM_JOB_UID=69294
SLURM_JOB_PARTITION=cpu
SLURM_JOB_USER=ibarbier
SLURM_NPROCS=1
SLURM_SUBMIT_HOST=login.curnagl
SLURM_JOB_ACCOUNT=yschaerl_snsf_2017
SLURM_GTIDS=0
SLURM_JOB_ID=5983318
SLURM_LOCALID=0
Traceback (most recent call last):
  File "plot.py", line 339, in <module>
    par_plot(pdf,filename,i,meq.parlist,namelist)
  File "plot.py", line 159, in par_plot
    plt.savefig(name+"/plot/par/"+nb+'_par_plot.png', bbox_inches='tight', dpi=300)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/pyplot.py", line 859, in savefig
    res = fig.savefig(*args, **kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/figure.py", line 2311, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/backend_bases.py", line 2217, in print_figure
    **kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/backend_bases.py", line 1639, in wrapper
    return func(*args, **kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/backends/backend_agg.py", line 512, in print_png
    dpi=self.figure.dpi, metadata=metadata, pil_kwargs=pil_kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/matplotlib/image.py", line 1611, in imsave
    image.save(fname, **pil_kwargs)
  File "/users/ibarbier/myfirstvenv/lib64/python3.6/site-packages/PIL/Image.py", line 2169, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: 'ACDC_X2_Kara2/plot/par/final_par_plot.png'
